### DISCUSSION TITLE
Show `Key Count` in mania's beatmap difficult name

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#19605] [Ideas]
In the current lazer version, you can only view `Key Count` in the details.
For stable version, it will be displayed before the difficulty name

stable:
![image](https://user-images.githubusercontent.com/34775378/183250147-62149ed0-ef3f-4456-96a3-411e6fe97d93.png)

lazer:
![image](https://user-images.githubusercontent.com/34775378/183250092-37537156-de1a-48af-88ed-e80f2bdd8a5a.png)

**Comment by nagi-desuuu** [comment]
Frenzibyte mentioned that there will be a song select redesign on the near future...
(I forgot the pull request/issue that references this so if anyone here knows it, please put it for cd's reference.)
